hudi-common/src/main/java/org/apache/hudi/common/model/HoodieFileFormat.java

```diff
@@ -22,7 +22,7 @@
  * Hoodie file format.
  */
 public enum HoodieFileFormat {
-  PARQUET(".parquet"), HOODIE_LOG(".log");
+  PARQUET(".parquet"), HOODIE_LOG(".log"), ORC(".orc");
```

Comment:
again, this can happen in a PR, that actually adds ORC support.. not now ?>

Reply:
Reasonable, done